src/auth/digest/Config.cc

```diff
@@ -774,6 +771,7 @@ Auth::Digest::Config::decode(char const *proxy_auth, const HttpRequest *request,
         }
 
         /* find type */
+        static LookupTable<http_digest_attr_type> DigestFieldsLookupTable(DIGEST_INVALID_ATTR, DigestAttrs);
```

Comment:
While unlikely, this local static might be destroyed _before_ the last call to this function in future Squid code. Thus, this change does not fully address the problem identified by Coverity. Consider using the safer pattern from the "No new globals" rule instead.

Reply:
ok